wapitiCore/report/htmlreportgenerator.py

```diff
@@ -66,9 +66,9 @@ def generate_report(self, output_path):
             output_encoding="utf-8"
         )
 
-        filename = "{}_{}.html".format(
-            urlparse(self._infos["target"]).netloc.replace(":", "_"),
-            time.strftime("%m%d%Y_%H%M", self._date)
+        filename = (
+            f"{urlparse(self._infos['target']).netloc.replace(':', '_')}_"
+            f"{time.strftime('%m%d%Y_%H%M', self._date)}.html"
```

Comment:
This is less readable, could you use intermediate variables with well-chosen names before the format string?